supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -324,12 +324,22 @@ There is no functional difference between the first server that was installed an
 
 [[ingress]]
 ==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The `Ingress` resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller (`IngressController`) should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
```

Comment:
From the context, it's not clear whether the Ingress API object and the Ingress resource are two different entities or the same entity.

Reply:
@mportman12 Ingress API object and Ingress resource are the same thing, but I don't think interchanging them in this definition does any good. Probably best to just stick with Ingress object. I also think more can be done in this definition to draw the connection between the API object, the Ingress Controller, and the Ingress Operator while we're at giving this a rework. I used the context here to help clarify this definition: https://docs.openshift.com/container-platform/4.12/networking/ingress-operator.html#nw-installation-ingress-config-asset_configuring-ingress